src/freenet/support/MultiValueTable.java

```diff
@@ -73,7 +75,7 @@ public boolean containsElement(K key, V value) {
 			v = table.get(key);
 		}
         return (v == null ?
-                new LimitedEnumeration<V>(null) :
+                new EmptyEnumeration<V>() :
```

Comment:
When we get to updating dependencies I would recommend simply using one of Guava’s excellent (and well-tested) `Multimap` implementations.